### DISCUSSION TITLE
Allow codelens method references to surface for Python

### DISCUSSION THREAD

**Discussion by lkupkeklaviyo** [microsoft/pylance-release#4740] [Enhancement]
I would like the ability to see method references in python like it has for Typescript, JS, and C#. 

I know this ticket was filed originally and the conversation was locked but this was back in 2017 & 2018. I was wondering what the teams thoughts are now that more people are using Python with VSCode.

Original issue ticket: https://github.com/Microsoft/vscode-python/issues/115

**Comment by heejaechang** [comment]
for us to support this, we need to move `find all reference` to `BG` and make `BG queue` `priority queue` so we can order `tasks` such as `checker`, `code lens` and `semantic coloring` vs `explicitly find all references/rename`

**Comment by omicadin** [comment]
hello, any news on this? I can't seem to find anything regarding this taht is "newer". All i see is old posts with no answers

**Reply by rchiodo** [reply]
There's been no work done on this yet as it's rather far down the priority list with only two up votes.
